src/main/resources/application.yml

```diff
@@ -19,18 +19,45 @@ spring:
 ---
 spring:
   profiles: local
+  jpa:
+    hibernate:
+      ddl-auto: create-drop
+    properties:
+      hibernate:
+        dialect: svc.dialects.ImprovedHSQLDialect
+  datasource:
+    data: classpath*:hsql/*.sql
```

Comment:
Maybe this is an alternative for the `platform` attribute. Either way, you don't need that datasource in the `LocalProfile` class.

Reply:
Correct.  I removed the datasource in the LocalProfile class.  This datasource.data will now read in all ".sql" files under the hsql directory in resources and load that data.